docs/general/emulator-support-and-issues.md

```diff
@@ -170,6 +170,7 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - ❌ _Not supported_ - needs hashing method and memory map
 - ❌ libretro core: **gw**
   - Does not expose memory
+  - Uses recreations (by MADrigal) instead of actual Game & Watch ROMs
```

Comment:
We can probably remove this too - this seems like a libretro implementation detail.